pyamg/krylov/_fgmres.py

```diff
@@ -108,6 +110,15 @@ def fgmres(A, b, x0=None, tol=1e-5,
        http://www-users.cs.umn.edu/~saad/books.html
 
     """
+    if restrt is None:
+        restrt = restart
```

Comment:
can this be a pass, and not restrt = restart?

Reply:
Good catch; I think it should be flipped.